index.html

```diff
@@ -3484,6 +3484,31 @@ <h3>Media Types</h3>
 those other syntaxes to be transformed to the `application/vc+ld+json`
 media type.
         </p>
+
+        <p>
+Additional media type transformation specifications can be
+defined to provide implementers with further options for producing objects
+that can be mapped to the base media type.
+        </p>
+        <p>
+Specifications are registered with the [[?VC-SPECS]] and
+MUST result in a transformed, concrete serialization
+that is objectively testable via a conformance test suite.
```

Comment:
It's ambiguous what you're saying should be testable.  Is it the transformed representation that is tested (in which case a VC Data Model test suite could be used).  Or are you also asking for testing of the additional media type representation?

Reply:
It's also not clear to me what the "MUST result" language means.  The day 3 resolution says that a transformation must exist.  It doesn't say that particularly deployments must use transformations.

Reply:
```
Specifications are registered with the [[?VC-SPECS]] and
MUST result in a transformed, concrete serialization
that is objectively testable via a conformance test suite.
```

Informative reference, followed by a MUST feels a bit off here.

If this is guidance to the registry maintainers, to gate entries I think this goes against our intention to keep the "vc specs directory", lightweight and not a registry.

Reply:
> It's ambiguous what you're saying should be testable. Is it the transformed representation that is tested (in which case a VC Data Model test suite could be used). Or are you also asking for testing of the additional media type representation?

The transformed representation should be testable against the VC Data Model test suite, yes.